### ISSUE TITLE
Old heading for rich content subsection

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#534]
In the guidelines, section 4.6.2 still uses the old heading:

> [Display techniques for charts, diagrams, and formulas support](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/guidelines/#cdf-techniques)

This should be "Display techniques for rich content". The links to the techniques also appear to use the old naming.
